### PR TITLE
Fix viewing issue on different devices in Featured Article section.

### LINKED ISSUE #1402: Issues when we open the Featured Articles section on different devices.
**Describe the bug**
There are some bugs when we open the featured articles section on different devices.
( Tab screen > 8.5-inch )

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to https://sefglobal.org/scholarx/2022/
2. Go to the Featured article section
3. View the section using different screen sizes.

**Expected behavior**
**I believe it is good to use two rows to show both read time and the author of the article.**

**Screenshots**
Screenshot 2022-11-20 191139gggg

Screenshot 2022-11-20 192508
Screenshot 2022-11-20 193508

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by janithlahirukariyawasam**
## Purpose

The purpose of this PR is to fix #1402

## Goals

To view the **Featured Article** section without any issue on any devices.

## Approach

Well, I tried different ways to solve this issue. But I believe it is good to use two rows to both Read time and Author section.
With that, You can view the featured article section on any device without any issues.
Next thing is viewers can easily understand the Author and Reading time without facing issues. (easy readability) 

### Screenshots

image

  
### Preview Link

https://pr-1405-sef-site.surge.sh/

##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by YoshithaRathnayake** [comment]
I think it is good if the read time goes down and name goes up. How about that? @anjula-sack @Piumal1999 @jayasanka-sack

**Comment by janithlahirukariyawasam** [comment]
Something like this? @YoshithaRathnayake 
WhatsApp Image 2022-11-21 at 08 10 47

**Comment by anjula-sack** [comment]
Shall we just get rid of the reading time? @jayasanka-sack @Piumal1999 @miniskynet @Madhawa97

**Comment by Piumal1999** [comment]
> Shall we just get rid of the reading time? @jayasanka-sack @Piumal1999 @miniskynet @Madhawa97

Yeah, that would be better. 
Btw, is it a hardcoded value?

**Comment by anjula-sack** [comment]
> Member

It's another field in the google sheet @Piumal1999

**Comment by janithlahirukariyawasam** [comment]
@anjula-sack @Piumal1999 What should I do guys? May I remove the Reading time?

**Comment by YoshithaRathnayake** [comment]
> > Member
> 
> It's another field in the google sheet @Piumal1999

So then shall I change it on the google sheet that we made for this @anjula-sack

**Comment by anjula-sack** [comment]
> @anjula-sack @Piumal1999 What should I do guys? May I remove the Reading time?

yeah let's remove it @janithlahirukariyawasam

**Comment by anjula-sack** [comment]
> > > Member
> > 
> > 
> > It's another field in the google sheet @Piumal1999
> 
> So then shall I change it on the google sheet that we made for this @anjula-sack

yep

**Comment by YoshithaRathnayake** [comment]
> yep

That means remove the reading time column right? @anjula-sack

**Comment by anjula-sack** [comment]
> > yep
> 
> That means remove the reading time column right? @anjula-sack

yeah let's wait until this PR is merged

**Comment by janithlahirukariyawasam** [comment]
image

I commented the reading time section.

File: scholarx/2022/index.html
Comment (anjula-sack) [inline scholarx/2022/index.html]: Let's just remove this. @janithlahirukariyawasam

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: scholarx/2022/index.html
Comment (janithlahirukariyawasam) [reply scholarx/2022/index.html]: Done

File: scholarx/2022/index.html
Comment (anjula-sack) [inline scholarx/2022/index.html]: Can you remove this line?
```suggestion
```

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: scholarx/2022/index.html
Comment (janithlahirukariyawasam) [reply scholarx/2022/index.html]: Sorry @anjula-sack

**Comment by YoshithaRathnayake** [comment]
> > > yep
> > 
> > 
> > That means remove the reading time column right? @anjula-sack
> 
> yeah let's wait until this PR is merged

Sure

File: scholarx/2022/index.html
Comment (anjula-sack) [reply scholarx/2022/index.html]: No worries

**Review by anjula-sack — APPROVED** [review]
Thank you! @janithlahirukariyawasam

**Review by YoshithaRathnayake — APPROVED** [review]

**Pull request merged**
